modules/library/src/main/scala/zio/elasticsearch/query/Queries.scala

```diff
@@ -121,6 +121,41 @@ private[elasticsearch] final case class Match[S, A: ElasticPrimitive](field: Str
   }
 }
 
+sealed trait HasParentQuery[S] extends ElasticQuery[S] with HasIgnoreUnmapped[HasParentQuery[S]] {
+  def withScore(value: Boolean): HasParentQuery[S]
+
+  def withScoreTrue: HasParentQuery[S] = withScore(true)
```

Comment:
1. Put `final def` here. 
2. Put `withScoreFalse` then `withScoreTrue`.
3. Should we put `withScore(...)` in the new line?

Reply:
3. We did not do it so far.